setup.py

```diff
@@ -72,7 +72,7 @@ def get_cryptography_version():
         "aiohttp<4.0; python_version<'3.7'",  # to support python3.6 (Centos 7)
         "aiohttp; python_version>='3.7'",
         get_cryptography_version(),
-        "CloudStackAIO>=0.0.8",
+        "CloudStackAIO",
```

Comment:
With the latest `COBalD` release `CloudStackAIO<0.0.8` will not work anymore.

Reply:
Strange, I did not remove this. Apparently my master wasn't up to date. Thanks for the comment! :)